charts/operator/deployment.go

```diff
@@ -230,6 +249,60 @@ func operatorPodVolumes(dot *helmette.Dot) []corev1.Volume {
 			},
 		},
 	}
+
+	if values.ServiceAccount.Create && !ptr.Deref(values.ServiceAccount.AutomountServiceAccountToken, false) {
+		vol = append(vol, kubeTokenAPIVolume(ServiceAccountVolumeName))
+	}
+
+	return vol
+}
+
+// It's slightly changed variance of
```

Comment:
nit:
```suggestion
// kubeTokenAPIVolume is a slightly changed variant of
```

Could you note what's changed or mention that divergences are commented in the body for clarity?

Reply:
Done